CHANGELOG.md

```diff
@@ -5,7 +5,13 @@ Multitenant AWS Dynamo supports the [AWS Dynamo Java API](http://docs.aws.amazon
   
 You can write your application code against the Amazon DynamoDB interface as you would for any other application.  The implementation will manage storage of data by tenant.
 
-## 0.9.48 (July 16, 2019)
+## 0.9.47 (July 25, 2019)
+
+* Fix paging for cross-tenant scans
+* Emit metric for age of evicted stream cache entries
```

Comment:
hyphenate "stream cache"

Reply:
Why is that hyphenated? Maybe replace with `StreamsRecordCache`?

Reply:
I like that even better than hyphenating